pkg/services/store/entity/entity.proto

```diff
@@ -320,10 +332,10 @@ message EntityListResponse {
 //-----------------------------------------------
 
 message EntityWatchRequest {
-  // Timestamp of last changes. Empty will default to
+  // ResourceVersion of last changes. Empty will default to full history
   int64 since = 1;
```

Comment:
AFAIK we can rename this without breaking backwards-compatibility, since internally protobuf relies on field ordinals, rather than names.